xml/depl_nodes.xml

```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
    </varlistentry>
   </variablelist>
 
+<bridgehead renderas="sect3"><guimenu>monasca: settings for libvirt and &ceph;
+monitoring</guimenu>
+  </bridgehead>
+  <variablelist>
+   <varlistentry>
+    <term>
+     monitor_libvirt
+    </term>
+    <listitem>
+     <para>
+      The global switch for toggling libvirt monitoring. If set to
+      <parameter>true</parameter>, libvirt metrics will be gathered on all
+      libvirt based &compnode;s. This setting is available in the &crow; UI.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     monitor_ceph
+    </term>
+    <listitem>
+     <para>
+      The global switch for toggling &ceph; monitoring. If set to
+      <parameter>true</parameter>, &ceph; metrics will be gathered on all
+      &ceph;-based &compnode;s. This setting is available in &crow; UI. If the
+      &ceph; cluster has been set up independently, &crow; ignores this setting.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     cache_dir
+    </term>
+    <listitem>
+     <para>
+      The directory where monasca-agent will locally cache various metadata
+      about locally running VMs on each &compnode;.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     customer_metadata
+    </term>
+    <listitem>
+     <para>
+      Specifies the list of instance metadata keys to be included as dimensions
+      with customer metrics. This is useful for providing more information
+      about an instance.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     disk_collection_period
+    </term>
+    <listitem>
+     <para>
+      Specifies a minimum interval in seconds for collecting disk metrics.
+      Increase this value to reduce I/O load. If the value is lower than the
+      global agent collection period (<option>check_frequency</option>), it
+      will be ignored in favor of the global collection period.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     max_ping_concurrency
+    </term>
+    <listitem>
+     <para>
+      Specifies the number of ping command processes to run concurrently when
+      determining whether the VM is reachable. This should be set to a value
+      that allows the plug-in to finish within the agent's collection period,
+      even if there is a networking issue. For example, if the expected number
+      of VMs per &compnode; is 40 and each VM has one IP address, then the
+      plug-in will take at least 40 seconds to do the ping checks in the
+      worst-case scenario where all pings fail (assuming the default timeout of
```

Comment:
Lines 6152 - 6155 is confusing to read. I thought it was a run on sentence at first but I realise the break. I wonder if this sounds any better:

"For example, if the expected number of VMs per &compnode; is 40 and each VM has one IP address, then the plug-in takes at least 40 seconds to perform ping checks in a worst-case scenario where all pings fail (assuming the default timeout of 1 second)."

Need other eyes, me thinks. Maybe I'm just tired? @dmpop